sdcm/provision/common/configuration_script.py

```diff
@@ -86,10 +86,11 @@ def _script_body(self) -> str:
         # 5. Make sure that whenever you use "cat <<EOF >>/file", make sure that EOF has no spaces in front of it
         script = ''
 
-        script += configure_syslogng_destination_conf(
-            host=self.syslog_host_port[0],
-            port=self.syslog_host_port[1],
-            throttle_per_second=SYSLOGNG_LOG_THROTTLE_PER_SECOND)
+        if self.logs_transport == 'syslog-ng':
+            script += configure_syslogng_destination_conf(
```

Comment:
Why not to add it under if statement on line 96?
https://github.com/scylladb/scylla-cluster-tests/blob/2223ba3cd8b5373e09d976b0b7984fbbd36208b5/sdcm/provision/common/configuration_script.py#L96

Reply:
We need to run `configure_syslogng_destination_conf` before this statement `script += self._skip_if_already_run()`
This `configure_syslogng_destination_conf` function prepares config for where logs will be sent. And if it is a new node then that skip check will have no effect and the remaining part of syslon-ng setup follows. If it is an existing node (when reusing the cluster), we need to just update logs destination and skip the syslog-ng setup.

This PR is just a quick fix. And the plan is to switch to syslog-ng for artifacts as well. So this and other `logs_transport` related ifs will disappear from this class.